### DISCUSSION TITLE
Can't get padloc to run

### DISCUSSION THREAD

**Discussion by BalakeV** [padloc/padloc#575] [Q&A] [answered]
I only recently started programming so apologies if I'm missing something blatantly obvious.
I downloaded the source code to play around with and see if I could learn from trying to understand/manipulate it but I can't get it to run using any method. Following the steps detailed in the readme.md:
> npm ci
npm start/npm run dev

nets me this error when trying either start or run dev:
>[webpack-cli] Error: something went wrong installing the "sharp" module`

No solution I could find online helped, not even uninstalling and reinstalling node and npm.

Attempting to use `docker-compose up` throws these errors:

> padloc-pwa-1   |   BREAKING CHANGE: No more changes should happen to Compilation.assets after sealing the compilation.
> padloc-nginx-1 |   exec /entrypoint.sh: no such file or directory

Same happens if I try to build the nginx dockerfile on its own or running the padloc/pwa image.

**Comment by BrunoBernardino** [comment, marked as answer]
Hey @BalakeV ! Regarding the `sharp` module, it's out-of-scope for us to really help (it's a natively-built dependency), but if you can provide your OS, npm, and node versions, we might be able to help.

Regarding `docker-compose up`, https://github.com/padloc/padloc/blob/main/docker-compose.yml needs a few things you might not have setup, would you mind trying a simpler setup with this file instead: https://github.com/padloc/padloc/blob/main/docs/examples/hosting/docker/basic/docker-compose.yml ?

**Comment by BalakeV** [comment]
Thanks, I'm running windows 10 x64, node version 16.17.0 and npm version 8.19.1. but it works now with the simpler docker-compose file.

**Reply by BrunoBernardino** [reply]
Ok, do you want to close this, or try `npm start` again to figure out what's the problem with `sharp`? Instead of `npm ci`, you can try `npm install` to see if that will allow the native building of it.
